exercises/tournament/tests/tournament.rs

```diff
@@ -44,12 +44,13 @@ fn a_different_team_can_win() {
 #[test]
 #[ignore]
 fn a_draw_is_one_point_each() {
-    let input = "Allegoric Alaskans;Blithering Badgers;draw";
-    let expected = "Team                           | MP |  W |  D |  L |  P\n".to_string()
-        + "Allegoric Alaskans             |  1 |  0 |  1 |  0 |  1\n"
-        + "Blithering Badgers             |  1 |  0 |  1 |  0 |  1";
+    let input = "Allegoric Alaskans;Blithering Badgers;loss\n".to_string()
+        + "Allegoric Alaskans;Blithering Badgers;win";
```

Comment:
Also you've taken out the `draw` input and replaced it (incorrectly) with one win apiece.

Reply:
Oops!